app/src/main/java/com/muratozturk/mova/ui/profile/ProfileFragment.kt

```diff
@@ -59,6 +61,13 @@ class ProfileFragment : Fragment(R.layout.fragment_profile) {
                     findNavController().navigate(action)
                 }
 
+                /**
+                 * Opening Privacy Policy Url on Device's Browser
+                 * @see [openWebPage]
+                 */
```

Comment:
No need to add this comment since it's self-explanatory. It adds unnessary maintenance.